### PR TITLE
Web client webpack again

### PR DESCRIPTION
**Pull request by gumbyscout**
Make sure you check the deploy.sh script to see that it's accurate, I didn't test it.

### CODE REVIEW

File: web/client/deploy.sh [hunk 1]
```diff
@@ -9,23 +9,22 @@ ts="$(date +%s)"
...
+npm install
+webpack
 
 mkdir -p "${dest}"
 
 tmpdest="$(mktemp -d)"
 
 # "content" is currently using an absolute directory, so don't copy it.
-cp -R index.html src/ jspm_packages/ jspm.config.js jspm.browser.js build.js build.js.map "${tmpdest}"
+cp -R index.html bundle.js bundle.js.map favicon.ico favicon-152.png "${tmpdest}"
```
Comment (nictuku) [inline web/client/deploy.sh]: are favicon.ico and favicon-152.png correct? I don't recall seeing them in this directory.

File: web/client/deploy.sh [hunk 2]
```diff
@@ -9,23 +9,22 @@ ts="$(date +%s)"
...
 
 # "content" is currently using an absolute directory
 # This changes the live stuff.
 # TODO: use versioning for the entire www directory.
 cp content/* "${rootdir}/content"
 
 sed -e "s#src=\"#src=\"assets/${ts}/#"  -i ${tmpdest}/*html
-sed -e "s#baseURL: \"/\"#baseURL: \"/assets/${ts}/\"#" -i "${tmpdest}/jspm.browser.js"
```
Comment (nictuku) [inline web/client/deploy.sh]: are paths all relative now?

**Comment by nictuku** [comment]
Tested in staging. It looks OK. But please see my comment about the favicon references in deploy.sh. Can you add a commit that moves the files appropriately?

**Comment by gumbyscout** [comment]
Currently, we don't have the favicon avaliable on prod, and they appear in web/client so they should be with the index.html and bundle.js at build.

File: web/client/deploy.sh (hunk 2 above)
Comment (gumbyscout) [inline web/client/deploy.sh]: The only path that is absolute, is the line in webpack.config.js
```
...
entry: [
    './src/main.js'
]
...
```

File: web/client/deploy.sh (hunk 1 above)
Comment (gumbyscout) [inline web/client/deploy.sh]: Currently, we don't have the favicon avaliable on prod, and they appear in web/client so they should be with the index.html and bundle.js at build. I.e. they should be correct.

**Pull request merged**